e2e/regression/guardrail.correction_range.spec.js

```diff
@@ -0,0 +1,50 @@
+const { LoopTest, screenName, target } = require('../../src/index');
+
+describe.skip('guard rail correction range', () => {
+    var loopTest;
+    it('should setup with correct configuration', async () => {
+        loopTest = await new LoopTest.Builder(target.tidepool)
+            .withSettings({ AddPumpSimulator: true })
+            .withStartScreen(screenName.settings)
+            .build();
+    });
+
+    // TODO: once CorrectionRanges uses accessibile pickers we should
+    // be able to launch loop once and then reuse the setup
+    it('maximum 180 can be set, warning', async () => {
+        await loopTest.settingsScreen.SetCorrectionRanges([{ time: '12:00 AM', min: '179', max: '180' }]);
+        //TODO assert on warning
+    });
+    it('maximum 121 can be set, warning', async () => {
+        await loopTest.settingsScreen.SetCorrectionRanges([{ time: '12:30 AM', min: '120', max: '121' }]);
+        //TODO assert on warning
+    });
+    it('maximum 120 can be set, no warning', async () => {
+        await loopTest.settingsScreen.SetCorrectionRanges([{ time: '1:00 AM', min: '119', max: '120' }]);
+    });
+    it('minimum 70 can be set, no warning', async () => {
+        await loopTest.settingsScreen.SetCorrectionRanges([{ time: '1:30 AM', min: '70', max: '180' }]);
+    });
+    it('minimum 69 can be set, warning', async () => {
+        await loopTest.settingsScreen.SetCorrectionRanges([{ time: '2:00 AM', min: '69', max: '180' }]);
```

Comment:
Does this case actually display two warnings? One for the min < 70 and one for max > 120? If so, might want that in the TODO so you remember.

Reply:
Actually, I would follow what you've done for the first three tests. Specifically, 179/180, 120/121, 119/120. 

Continue that in these tests.  Specifically, 69/70, 60/61.

Then add new tests to test warnings on both min and max. Specifically 69/121.

Finally, just for the sake of reading, I'd put the 181 test at the top so all of the tests are in order from maximum max to minimum min.